static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -118,6 +143,23 @@ class ProjectPerformance extends AsyncView<Props, State> {
     );
   };
 
+  handleThresholdsReset = () => {
+    const {projectId} = this.props.params;
+    const {organization} = this.props;
+
+    this.setState({
+      loading: true,
+    });
+
+    this.api.request(
+      `/projects/${organization.slug}/${projectId}/performance-issues/configure/`,
+      {
+        method: 'DELETE',
+        complete: () => this.fetchData(),
+      }
+    );
```

Comment:
We talked about this but I just wanted to comment here that I noticed this doesn't just reset the thresholds panel but all of the performance issue stuff. Including the issue occurrences flags and the admin options. But we agreed we could improve this in a new PR

Reply:
I added the reset as a put request for now. Also, added a todo [here](https://www.notion.so/sentry/Detector-Threshold-Configuration-f8cb07e7cceb42388cedb09ea05fc116?pvs=4#98f4af6b47f44ab295864886ec870976).